{"CVE Code": "CVE-2021-28676", "Description": "An issue was discovered in Pillow before 8.2.0. For FLI data, FliDecode did not properly check that the block advance was non-zero, potentially leading to an infinite loop on load.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2021-28676", "Vulnerability Categories": "N/A", "Affected Products": "\nFedoraproject\u00bbFedora\u00bbVersion:33cpe:2.3:o:fedoraproject:fedora:33:*:*:*:*:*:*:*Matching versions\nPython\u00bbPillowVersions  before (<) 8.2.0cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*Matching versions", "EPSS Score": "0.42% Probability of exploitation activity in the next 30 days\n~ 74 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2024-12-17  0.35%  0.42%  +0.07 \n 2  2024-02-16  0.31%  0.35%  +0.04 \n 3  2024-01-07  0.30%  0.31%  +0.01 \n 4  2023-12-23  0.43%  0.30%  -0.14 \n 5  2023-12-12  0.42%  0.43%  +0.02 \n 6  2023-11-17  0.23%  0.42%  +0.19 \n 7  2023-11-08  0.23%  0.23%  0.00 \n 8  2023-10-04  0.16%  0.23%  +0.06 \n 9  2023-03-07  1.02%  0.16%  -0.85 ", "CVSS Scores": "5.0", "CWE": [{"CWE_ID": "CWE-835 Loop with Unreachable Exit Condition ('Infinite Loop')", "CWE_link": "/cwe-details/835/Loop-with-Unreachable-Exit-Condition-Infinite-Loop-.html"}], "Reference": [{"ref_link": "https://github.com/python-pillow/Pillow/commit/bb6c11fb889e6c11b0ee122b828132ee763b5856", "ref_desc": "Fix FLI DOS -- CVE-2021-28676 \u00b7 python-pillow/Pillow@bb6c11f \u00b7 GitHubPatch", "ref_cve_link": "N/A"}, {"ref_link": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/MQHA5HAIBOYI3R6HDWCLAGFTIQP767FL/", "ref_desc": "[SECURITY] Fedora 33 Update: mingw-python-pillow-7.2.0-6.fc33 - package-announce - Fedora Mailing-ListsMailing List", "ref_cve_link": "N/A"}, {"ref_link": "https://lists.debian.org/debian-lts-announce/2021/07/msg00018.html", "ref_desc": "[SECURITY] [DLA 2716-1] pillow security updateThird Party AdvisoryCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/4iPqfVbxWvaMO4OV941-RjLHfAA.html"}, {"ref_link": "https://pillow.readthedocs.io/en/stable/releasenotes/8.2.0.html#cve-2021-28676-fix-fli-dos", "ref_desc": "8.2.0 \u2014 Pillow (PIL Fork) 8.2.0 documentationRelease Notes;Third Party Advisory", "ref_cve_link": "N/A"}, {"ref_link": "https://github.com/python-pillow/Pillow/pull/5377", "ref_desc": "Security fixes for 8.2.0 by hugovk \u00b7 Pull Request #5377 \u00b7 python-pillow/Pillow \u00b7 GitHubThird Party AdvisoryCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/Re_S1hIp6dg3UUV8g5dM1Y5W-L4.html"}, {"ref_link": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MQHA5HAIBOYI3R6HDWCLAGFTIQP767FL/", "ref_desc": "[SECURITY] Fedora 33 Update: mingw-python-pillow-7.2.0-6.fc33 - package-announce - Fedora Mailing-ListsMailing List;Third Party AdvisoryCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/v8tOzxB5HjtGTBfd6LATaB4sl_g.html"}, {"ref_link": "https://security.gentoo.org/glsa/202107-33", "ref_desc": "Pillow: Multiple vulnerabilities (GLSA 202107-33) \u2014 Gentoo securityThird Party AdvisoryCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/vimQGyiGjUm9cmiWZIR07UQ3jmM.html"}]}